packages/react-app/src/lib/constants.js

```diff
@@ -14,10 +18,52 @@ export const POLLING_INTERVAL =
 export const DEFAULT_BRIDGE_DIRECTION =
   process.env.REACT_APP_DEFAULT_BRIDGE_DIRECTION || ETH_XDAI_BRIDGE;
 
+export const NATIVE_CURRENCY_SYBMOLS = ['ETH', 'KETH', 'BNB'];
+export const NATIVE_CURRENCY_CHAIN_IDS = [1, 42, 56];
```

Comment:
Could we remove these constants and have these symbols or if native currency is enabled as part of the `lib/networks` file? 
Also need to disable showing `ETH` (native currency) for mainnet until the helper contracts are reviewed and deployed.